tutorial/errors.po

```diff
@@ -29,6 +30,8 @@ msgid ""
 "tried out the examples you have probably seen some.  There are (at least) "
 "two distinguishable kinds of errors: *syntax errors* and *exceptions*."
 msgstr ""
+"到目前為止錯誤訊息還沒有提到，但如果你嘗試這些範例，你可能會發現一些問題。有"
+"(至少)兩種明顯的錯誤分別是*語法錯誤*和*例外*。"
```

Comment:
The parentheses here should be full-width.  The stars for marking the italic texts should have "un-spaced separator" put before them, as described in the translation guide.

Also, the word "分別" could be ambiguous, it could mean both "distinguishing" or "namely blah and blah", I think it would be better if some proper punctuation is attached to make the meaning unambiguous.